sale_order_line_sequence/__manifest__.py

```diff
@@ -1,12 +1,16 @@
 # Copyright 2017 Eficent Business and IT Consulting Services S.L.
 # Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2022-TODAY Openforce srls
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Sale Order Line Sequence",
     "summary": "Propagates SO line sequence to invoices and stock picking.",
-    "version": "15.0.1.0.0",
-    "author": "Eficent, Serpent CS, Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Eficent, "
+    "Serpent CS, "
+    "Openforce srls, "
```

Comment:
Migration does not allow you to put yourself as author.

Instead put a line in readme/CONTRIBUTORS.rst

Reply:
Ok sorry. Iḿ fix this immediatly

Reply:
@rousseldenis i need to remove the copyright line added?